contracts/utils/test/lib_eip712.ts

```diff
@@ -73,6 +54,32 @@ describe('LibEIP712', () => {
         await blockchainLifecycle.revertAsync();
     });
 
+    /**
```

Comment:
Can we move this to right before the `_hashEIP712Message` tests and put `testHashEIP712DomainAsync()` in this place?